x/evm/keeper/keeper_test.go

```diff
@@ -228,3 +201,45 @@ func (suite *KeeperTestSuite) DeployTestContract(t require.TestingT, owner commo
 func TestKeeperTestSuite(t *testing.T) {
 	suite.Run(t, new(KeeperTestSuite))
 }
+
+func (suite *KeeperTestSuite) TestBankHooks() {
```

Comment:
this test could be a dummy EVM hook and for the follow up PR we can add this on the new module that implements this hook

Reply:
done